src/components/ha-target-picker.ts

```diff
@@ -250,11 +251,10 @@ export class HaTargetPicker extends SubscribeMixin(LitElement) {
 
   private async _showPicker(ev) {
     this._addMode = ev.currentTarget.type;
-    await this.updateComplete;
-    setTimeout(() => {
-      this._inputElement?.open();
-      this._inputElement?.focus();
-    }, 0);
+    await nextRender();
```

Comment:
this should stay `updateComplete`, as we wait for the `_inputElement` to be rendered